client/control/components/Fields.js

```diff
@@ -0,0 +1,48 @@
+import React, { PropTypes as pt } from 'react';
+import { Field } from 'redux-form';
+
+/**
+ * redux-form Field component that wraps the form input in a label and error
+ * message container.
+ *
+ * See buildControlField for supported props.
+ */
+export function ControlField({ component, ...args }) {
+  return <Field component={buildControlField} InputComponent={component} {...args} />;
+}
+ControlField.propTypes = {
+  component: pt.oneOfType([pt.func, pt.string]).isRequired,
+};
+
+/**
+ * Builds the React component that is rendered for ControlField.
+ */
+export function buildControlField({
+  input,
+  meta: { error },
+  label,
+  className = '',
+  InputComponent,
+  children,
+  ...args // eslint-disable-line comma-dangle
```

Comment:
Yeah. The problem is that it's syntactically invalid to have a comma after the spread argument, but the rule demands commas after all arguments when parameters are split up like this. So I had to disable the rule.